src/effects/effectchainslot.cpp

```diff
@@ -68,6 +68,12 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
 
     connect(&m_channelStatusMapper, SIGNAL(mapped(const QString&)),
             this, SLOT(slotChannelStatusChanged(const QString&)));
+
+    // It is up to skins and controller mappings to do anything useful with this.
+    m_pControlExpanded = new ControlPushButton(ConfigKey(m_group, "expanded"));
```

Comment:
> Ah OK I understand. Than this is basically the "mode" switch on a vestax vci-400 and the "FX Mode" switch on Pioneer DDJ-SR & RR right?

Correct.

> In addition, I think we need a focus control, that allows to highlight the effect in the GUI that is currently controlled in "single mode". On the Pioneer DDJ-SR & RR it is called "FX Select"

That would be great, but AFAIK there isn't any way to really make use of that in skins yet. It would be great to be able to show a box around the selected effect [like Ableton can do with the active areas in its Live View](https://www.youtube.com/watch?v=IAWy2TVtXo8&t=1m55s).